app/Http/Controllers/Admin/UserController.php

```diff
@@ -41,4 +46,29 @@ public function show()
     {
         return view('backend.users.show');
     }
+
+    /**
+     * Change role of user when click button.
+     *
+     * @param Id $id Id of user changed
+     *
+     * @return mixed
+     */
+    public function changerole($id)
```

Comment:
use changeRole instead of changerole.
http://172.16.110.17/attachments/article_attachment/attachment/67/c92bf26895.pdf